driver/src/price_finding/mod.rs

```diff
@@ -19,7 +19,7 @@ pub fn create_price_finder(
     } else {
         info!(
             "Using optimization price finder with the args {:?}",
-            solver_type.to_args()
+            solver_type.to_args("RESULT_FOLDER", "INPUT_FOLDER", String::from("TIME_LIMIT"))
```

Comment:
Its werd to see `String::from` on only one of these input parameters. Was there a reason for this?